ui/src/components/AlertNavbarUpdaterComponent.tsx

```diff
@@ -0,0 +1,189 @@
+import { useEffect } from 'react';
+import AlertProvider, { useAlerts } from '../containers/AlertProvider';
+import { Alert } from '../services/alertUtils';
+import FederatedIntlProvider from '../containers/IntlProvider';
+import {
+  AppConfigProvider,
+  AppConfigProviderWithoutRedux,
+  useConfig,
+} from '../FederableApp';
+import ConfigProvider from '../containers/ConfigProvider';
+
+type Notification = {
+  id: string;
+  title: string;
+  description: string;
+  severity: 'critical' | 'warning' | 'info';
+  redirectUrl: string;
+  createdOn: Date;
+};
+
+const WATCHDOG_ALERT_NAME = 'Watchdog';
+const CRITICAL_NOTIFICATION_ID = 'CriticalNotification';
+const WARNING_NOTIFICATION_ID = 'WarningNotification';
+
+const LOCAL_STORAGE_All_ALERTS_ID = 'alertIDs';
+
+export default function AlertNavbarUpdaterComponent({
+  publishNotification,
+  unPublishNotification,
+}: {
+  publishNotification: (notification: Notification) => void;
+  unPublishNotification: (id: string) => void;
+}) {
+  return (
+    <FederatedIntlProvider>
+      <ConfigProvider>
+        <AppConfigProvider
+          componentWithInjectedImports={AppConfigProviderWithoutRedux}
+        >
+          <AlertProvider>
+            <AlertNavbarUpdaterComponentInteral
+              publishNotification={publishNotification}
+              unPublishNotification={unPublishNotification}
+            />
+          </AlertProvider>
+        </AppConfigProvider>
+      </ConfigProvider>
+    </FederatedIntlProvider>
+  );
+}
+
+function publishCriticalNotification(
+  publishNotification: (notification: Notification) => void,
+  warningAlerts: any,
+  criticalAlerts: any,
```

Comment:
`any` cause storms every year, we need to take care of them 😄 
```suggestion
  warningAlerts: TYPE_ME,
  criticalAlerts: TYPE_ME,
```